token/driver/request.go

```diff
@@ -154,8 +160,28 @@ func (m *TokenRequestMetadata) Bytes() ([]byte, error) {
 			ExtraSigners:       transfer.ExtraSigners,
 		}
 	}
+
+	// marshal issues
+	issues := make([]IssueMetadataSer, len(m.Issues))
+	for i, issue := range m.Issues {
+		TokenIDs := make([]TokenIDSer, len(issue.TokenIDs))
+		for j, tokenID := range issue.TokenIDs {
+			TokenIDs[j].TxId = tokenID.TxId
+			TokenIDs[j].Index = int(tokenID.Index)
```

Comment:
Is there a problem with the serialization of uint64?

Reply:
yes, asn1 does not support it